guac_actix/Cargo.toml

```diff
@@ -4,11 +4,14 @@ version = "0.1.0"
 authors = ["Ben Wang <wangben3@gmail.com>"]
 
 [dependencies]
-actix = "0.5.7"
-actix-web = "0.6.7"
+actix = { git = "https://github.com/kingoflolz/actix.git", branch = "althea-mesh" }
```

Comment:
how's getting this merged upstream going?

Reply:
working on it, think we've worked out how to handle global actors which have not been started, still need to create pr